### ISSUE TITLE
Some parts of the plot are hidden

### ISSUE THREAD

**Issue by libremente** [italia/covid19-dashboard-vaccini#24]
As shown below: 
![Schermata da 2021-01-05 19-42-26](https://user-images.githubusercontent.com/25111463/103685859-3484c980-4f8e-11eb-959e-e2905e9ad5b2.png)
some parts of the plot are hidden.

**Comment by libremente** [comment]
Furthermore, the plot title should probably be removed since it's the same as the section one.

**Comment by sebbalex** [comment]
> Furthermore, the plot title should probably be removed since it's the same as the section one.

that is the x axis title, however we can remove it since it is repetitive.

**Comment by libremente** [comment]
Yeah but I believe we should either change the label (and the font face?) or remove it. Isn't it self explanatory?

**Comment by libremente** [comment]
This has been fixed on the official repo, closing
